setup.cfg

```diff
@@ -1,3 +1,16 @@
 [bdist_wheel]
 # code is written to work on both Python 2 and Python 3
-universal=1
+universal = 1
+
+[pep8]
+max-line-length = 119
```

Comment:
The reason we have set the limit to 119 is because we have followed [Django's coding guidelines](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).